src/args/argument_list.go

```diff
@@ -15,6 +17,7 @@ type ArgumentList struct {
 	Hostname               string `default:"localhost" help:"The agent node Hostname or IP address to connect to"`
 	Port                   string `default:"8500" help:"Port to connect to agent node"`
 	Token                  string `default:"" help:"ACL Token if token authentication is enabled"`
+	Timeout                string `default:"0s" help:"Timeout for an API call"`
```

Comment:
we might consider to add a default timeout , perhaps related to the default integration interval 30s